File: python/encrypt.py
'''
import random
a = random.randint(3, 9999)
b= random.randint(3, 83901)
c= a*b
print("First Number= ", a)
print("Second Number= ", b)
print("Sum of a & b Number= ", c)
'''
'''
IMPORTANT CODE FOR PRIME
a= int(input("Enter a Num= "))
factor= 0
for i in range (1, a+1):
	if a % i ==0:
		factor=factor+1
		
if factor > 2:		
	print("Not prime")
else:
	print("Prime")
'''
def generate_prime(digits):
	a = random.randint(10**digits, 10**(digits+1))
	while True:
		factor= 0
		for i in range (1, a+1):
			if a % i == 0:
				factor=factor+1
			
			
		if factor > 2:	
			print(a,"is not a prime no.")
			a = a+1
		else:
			print("Your Prime No. is ", a)
			return a


import random

a = generate_prime(7)			
b = generate_prime(7)

c= a*b
print("First Number= ", a)
print("Second Number= ", b)
print("Sum of a & b Number= ", c)

File: c/encrypt.c
#include <random.h>
#include <stdio.h>

long unsigned int generate_prime(long unsigned int digits)
{
	a = random.randint(10**digits, 10**(digits+1))
	while()
	{
		int factor= 0;
		for(int i=1; i<=a; a++)
			if (a % i == 0)
				factor=factor+1;
			
			
		if (factor > 2)	
			printf("%d is not a prime no.", a);
		else
		{
			print("Your Prime No. is %d", a);
			return a;
		}
	}
}

int main()
{
	long unsigned int a = generate_prime(7);
	long unsigned int b = generate_prime(7);
}
